indico/modules/events/timetable/client/js/DayTimetable.tsx

```diff
@@ -38,23 +38,33 @@ interface DayTimetableProps {
   entries: TopLevelEntry[];
 }
 
-function TopLevelEntries({dt, entries}: {dt: Moment; entries: TopLevelEntry[]}) {
+function TopLevelEntries({
+  dt,
+  entries,
+  eventId,
```

Comment:
Won't we already be able to get the eventId in some other way from the entry or something? Is it necessary to add eventId to `TopLevelEntries`?